doc/src/devdocs/offset-arrays.md

```diff
@@ -223,15 +189,23 @@ Base.reshape(A::AbstractArray, shape::Tuple{ZeroRange,Vararg{ZeroRange}}) = ...
 
 and you can `reshape` an array so that the result has custom indices.
 
-## Summary
+### For objects that mimic AbstractArray but are not subtypes
+
+`is_non1_indexed` has a specialized implementation for `AbstractArrays`
```

Comment:
`has_offset_axes` here and immediately below, and it no longer defaults to false